src/customize-pop/online-accounts.md

```diff
@@ -0,0 +1,44 @@
+# Online Accounts
+
+Settings to connect online accounts are located in `Settings` ➞ `Online Accounts`. Online Accounts settings provide single sign-on for online providers. Connected accounts can directly access core services like Mail, Calendar, Contacts, Maps, Photos, Files, and Music, without requiring sign-on through a web browser.
+
+---
+
+## Required Data Access
+
+This chart is also found on the [GNOME wiki](https://wiki.gnome.org/Projects/GnomeOnlineAccounts/Providers).
```

Comment:
Maybe say "An up-to-date version of this chart can also be found..." or some other phrasing that makes clear why you're linking out.

Reply:
Updated